seacrowd/sea_datasets/typhoon_yolanda_tweets/typhoon_yolanda_tweets.py

```diff
@@ -0,0 +1,139 @@
+
+_ROOT_URL = "https://raw.githubusercontent.com/imperialite/Philippine-Languages-Online-Corpora/master/Tweets/Annotated%20Yolanda/"
+_URLS = {"train": {-1: _ROOT_URL + "train/-1.txt", 0: _ROOT_URL + "train/0.txt", 1: _ROOT_URL + "train/1.txt"}, "test": {-1: _ROOT_URL + "test/-1.txt", 0: _ROOT_URL + "test/0.txt", 1: _ROOT_URL + "test/1.txt"}}
+
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]  # example: [Tasks.TRANSLATION, Tasks.NAMED_ENTITY_RECOGNITION, Tasks.RELATION_EXTRACTION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+# TODO: Name the dataset class to match the script name using CamelCase instead of snake_case
+class TyphoonYolandaTweets(datasets.GeneratorBasedBuilder):
+    """
+    The dataset contains annotated typhoon and disaster-related tweets in Filipino collected before, during, and
+    after one month of Typhoon Yolanda in 2013. The dataset has been annotated by an expert into three sentiment
+    categories: positive, negative, and neutral.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="typhoon_yolanda_tweets_source",
+            version=SOURCE_VERSION,
+            description="Typhoon Yolanda Tweets source schema",
+            schema="source",
+            subset_id="typhoon_yolanda_tweets",
+        ),
+        SEACrowdConfig(
+            name="typhoon_yolanda_tweets_seacrowd_text",
+            version=SEACROWD_VERSION,
+            description="Typhoon Yolanda Tweets SEACrowd schema",
+            schema="seacrowd_text",
+            subset_id="typhoon_yolanda_tweets",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "typhoon_yolanda_tweets_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "text": datasets.Value("string"),
+                    "label": datasets.Value("string"),
+                }
+            )
+        elif self.config.schema == "seacrowd_text":
+            features = schemas.text_features(["-1", "0", "1"])
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        emos = [-1, 0, 1]
+        # TODO: KEEP if your dataset is LOCAL; remove if NOT
+        if self.config.name == "typhoon_yolanda_tweets_source" or self.config.name == "typhoon_yolanda_tweets_seacrowd_text":
+            train_path = dl_manager.download_and_extract({emo: _URLS["train"][emo] for emo in emos})
+
+            test_path = dl_manager.download_and_extract({emo: _URLS["test"][emo] for emo in emos})
```

Comment:
remove TODO, remove extra new line
```suggestion
        emos = [-1, 0, 1]
        if self.config.name == "typhoon_yolanda_tweets_source" or self.config.name == "typhoon_yolanda_tweets_seacrowd_text":
            train_path = dl_manager.download_and_extract({emo: _URLS["train"][emo] for emo in emos})
            test_path = dl_manager.download_and_extract({emo: _URLS["test"][emo] for emo in emos})
```